app/src/androidTest/java/com/github/h3lp3rs/h3lp/HelpParametersActivityTest.kt

```diff
@@ -68,14 +47,18 @@ class HelpParametersActivityTest {
 
     @Before
     fun setUp() {
-        init()
+        initIntentAndCheckResponse()
+
         globalContext = getApplicationContext()
```

Comment:
Maybe these redundant setup functions can be added in the H3lpAppTest? Suggestion for future enhancements.